apps/site-projects/src/components/modules/Projects/Project/Project.js

```diff
@@ -74,159 +74,3 @@ const Project = ({ project, theme }) => {
 };
 
 export default withTheme(Project);
-
-// const Project = (props) => {
```

Comment:
Should this component be commented?

Reply:
looks like this code was refactored by @Enjoy2Live